rocky/tools/migrations/0039_update_permissions.py

```diff
@@ -0,0 +1,28 @@
+from django.db import migrations
+
+from tools.models import GROUP_CLIENT, GROUP_REDTEAM
+
+
+def add_group_permissions(apps, schema_editor):
+    Group = apps.get_model("auth", "Group")
+    Permission = apps.get_model("auth", "Permission")
+    try:
+        redteam = Group.objects.get(name=GROUP_REDTEAM)
+        redteam.permissions.add(Permission.objects.get(codename="can_mute_findings"))
+        redteam.permissions.add(Permission.objects.get(codename="can_view_katalogus_settings"))
+        redteam.permissions.add(Permission.objects.get(codename="can_set_katalogus_settings"))
+
+        clients = Group.objects.get(name=GROUP_CLIENT)
+        clients.permissions.add(Permission.objects.get(codename="can_scan_organization"))
+    except Group.DoesNotExist:
+        pass
```

Comment:
Can the `can_recalculate_bits` permission be added to the admin group in this migration? We forgot to do this earlier and if we add it here we don't need to add an extra migration for this.

Reply:
Done